src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -362,8 +381,10 @@ type private DiffixSum(summandType, aidsCount) =
       mergeContributions state.Positive srcState.Positive
       mergeContributions state.Negative srcState.Negative
 
-    member this.Final(aggContext, anonContext) =
+    member this.Final(aggContext, anonContext, outliersAggregrator) =
       let anonContext = unwrapAnonContext anonContext
+      let getPositiveState = fun aggregrator -> (castAggregator<DiffixSum> aggregrator).State.Positive
+      let getNegativeState = fun aggregrator -> (castAggregator<DiffixSum> aggregrator).State.Negative
```

Comment:
Typo in `aggregrator`

Reply:
Also saw it in a few other places with CTRL-F.

Reply:
Thanks, seems that my brain miss-fired so hard when it came to writing this word that it completely replaced the correct spelling in my head... 🙄